src/storage.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+export const addAnswerToStorage=(answer)=>{
+    let answers=getAnswersFromStorage();
+    answers.push(answer);
```

Comment:
Currently the code assumes that every time the next button is clicked, it is a new answer. That assumes there is no bug and it always goes right. I think it is better to also pass the index of the question to this function, so that you can store the answer on the right index in the array. If the `nextQuestion` function is then executed twice by accident, it still goes right.